python/setup.cfg

```diff
@@ -1,3 +1,11 @@
+[tox]
+envlist = py27,py36
+
+[testenv]
+commands = pytest {posargs:-vv}
+passenv = REDIS_HOST
+deps = ipdb
```

Comment:
Why is ipdb required? Is this left over from debugging?

Reply:
Yes, but ultimately it is needed anytime you want to debug a test, so I argue it makes sense to leave it. I'm open-minded. We don't have it in Crucible's CI tox env, so maybe we don't need it here.

Reply:
I'll remove it, we'll see :)